### PR TITLE
Feature/56

### PR DESCRIPTION
**Pull request by shivenduverma-sap**

### CODE REVIEW

**Comment by shivenduverma-sap** [comment]
TODO after review, before merge: 
  - use latest version of ocm-e2e-framework

File: .gitignore [hunk 1]
```diff
@@ -43,3 +43,4 @@ e2e/testdata/testSignedOCIRegistryComponents/podinfo/redis/manifests/*
 
 # ignore generated certificates
 hack/certs
+!hack/certs/README.md
```
Comment (Skarlso) [inline .gitignore:46]: The readme is already included in the repository. This should not be needed. 🤔

File: config/registry/kustomization.yaml [hunk 2]
```diff
@@ -5,5 +5,5 @@ resources:
   - service.yaml
 
 # Uncomment to enable HTTPS for the registry
-#patches:
-#- path: ./patches/add_registry_certificates.yaml
+patches:
```
Comment (Skarlso) [inline config/registry/kustomization.yaml:8]: This should not be enabled. For testing purposes Tilt patches the deployment.

File: controllers/localization_controller.go [hunk 3]
```diff
@@ -83,11 +83,15 @@ func (r *LocalizationReconciler) SetupWithManager(mgr ctrl.Manager) error {
...
+		var ns = ""
+
+		if loc.Spec.ConfigRef != nil && len(loc.Spec.ConfigRef.Namespace) != 0 {
+			ns = loc.Spec.ConfigRef.Namespace
+		}
+		if len(ns) == 0 {
 			ns = loc.GetNamespace()
 		}
-		return []string{fmt.Sprintf("%s/%s", ns, loc.Spec.ConfigRef.Name)}
+		return []string{fmt.Sprintf("%s", ns)}
```
Comment (Skarlso) [inline controllers/localization_controller.go:94]: This is missing the name here. This should be `name/namespace`.

File: controllers/localization_controller.go [hunk 4]
```diff
@@ -83,11 +83,15 @@ func (r *LocalizationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	if err := mgr.GetFieldIndexer().IndexField(context.TODO(), &v1alpha1.Localization{}, configKey, func(rawObj client.Object) []string {
 		loc := rawObj.(*v1alpha1.Localization)
-		var ns = loc.Spec.ConfigRef.Namespace
```
Comment (Skarlso) [inline controllers/localization_controller.go:86]: Adopting this change in https://github.com/open-component-model/ocm-controller/pull/250 so Gerald can't move forward with some demos.

**Comment by shivenduverma-sap** [comment]
Waiting for approval to merge

**Pull request merged**